changelog.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.10] - 2023-06-13
+### Changes
```

Comment:
I think we should mention the nginx/ruby/... version upgrades in `changelog.md` too

Reply:
Those upgrades are mention in the [v1.10 release](https://github.com/heroku/heroku-buildpack-nginx/releases/tag/v1.10).

Why do we keep a changelog separately from the GitHub releases? I will update this, but seems like a relic.

Reply:
Done: [https://github.com/heroku/heroku-buildpack-nginx/blob/main/changelog.md#110---2023-06-13](changelog.md)